src/games/strategy/engine/data/GameParser.java

```diff
@@ -56,6 +55,7 @@
   private static final Class<?>[] SETTER_ARGS = {String.class};
   private GameData data;
   private final Collection<SAXParseException> errorsSAX = new ArrayList<SAXParseException>();
+  final static public String dtdFileName = "game.dtd";
```

Comment:
:+1:  on refactoring this

nitpick: class level  final immutable variables get caps+underscores for their names. `DTD_FILE_NAME`